.github/workflows/holiday-generator/config.js

```diff
@@ -0,0 +1,84 @@
+import { cwd } from "node:process";
+import { join } from "node:path";
+
+export const SHOULD_LOG = process.env.LOG_ENABLED === "true";
+
+export const ICS_PATH = join(cwd(), "../../../app/src/main/assets");
+
+// country codes from https://github.com/commenthol/date-holidays?tab=readme-ov-file#supported-countries-states-regions
```

Comment:
https://www.thunderbird.net/en-US/calendar/holidays
https://www.officeholidays.com/subscribe/

you could also parse from any one from above? Asking so because I found them to up-to-date (had been using officeholidays well over 2 years)

Reply:
officeholidays is copyrighted, so we can't use it without violating the license. Thunderbird's calendar is AFAIR licensed as CC BY-SA, so we'd need to write that they are authors of these calendars and distribute them on the same license. TBH, I don't know if and how CC BY-SA is compatible with GPLv3, so using MIT-licensed date-holidays was the easiest way license-wise.

Reply:
Got it, did not think about licensing though 😅